### DISCUSSION TITLE
Batch Upscale Output as JPG?

### DISCUSSION THREAD

**Discussion by mxmx99** [upscayl/upscayl#377] [Releases]
I was wondering if its possible to get the images as a .jpg file after I do a batch upscale?
My images are all .jpg files and I choose JPG in the settings, still they come out as PNG.

**Comment by aaronliu0130** [comment]
From the FAQ:
> - Why can't batch upscaling output other formats such as JPGs instead of PNGs?
>  - This is due to an issue with REAL-ERSGAN that we can't fix currently tracked [here](https://github.com/upscayl/upscayl/issues/174).
